dandiapi/api/models/asset.py

```diff
@@ -218,10 +218,7 @@ def is_different_from(
         if self.path != path:
             return True
 
-        if self.metadata != metadata:
-            return True
-
-        return False
+        return self.metadata != metadata
```

Comment:
Please revert this change, as it is not connected to the PR's purpose.

(I believe the code was written this way for stylistic consistency with the previous `if` blocks, each of which tests a single condition.)

Reply:
I see now that this was done to avoid linting/formatting failures. But this was resolved in a different way, so merging master into this branch will resolve this problem.